newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -662,13 +716,52 @@ export default class ThemableEventsTree extends Component<
                 eventsHeightsCache={this.eventsHeightsCache}
                 selection={this.props.selection}
                 leftIndentWidth={depth * getIndentWidth(this.props.windowWidth)}
-                onAddNewInstruction={this.props.onAddNewInstruction}
-                onPasteInstructions={this.props.onPasteInstructions}
-                onMoveToInstruction={this.props.onMoveToInstruction}
-                onMoveToInstructionsList={this.props.onMoveToInstructionsList}
-                onInstructionClick={this.props.onInstructionClick}
-                onInstructionDoubleClick={this.props.onInstructionDoubleClick}
-                onParameterClick={this.props.onParameterClick}
+                onAddNewInstruction={instructionsListContext => {
+                  node.event &&
```

Comment:
You should be able to remove all these `node.event` since there is a check at the beginning of the function `if (!event) return null;`